### ISSUE TITLE
No link back to amo from the developers landing page.

### ISSUE THREAD

**Issue by muffinresearch** [mozilla/addons-server#4694]
There doesn't seem to be any way to navigate back to the main amo site from the developers page. This should be possible - we shouldn't require users to hack urls for navigation.

Maybe adding it to the footer would work but if that happens bear in mind the bottom footer is not shown on mobile viewports currently (see #4693)

**Comment by muffinresearch** [comment]
Duplicate of #4580
